### ISSUE TITLE
`pgweb`

### ISSUE THREAD

**Issue by nelsonic** [dwyl/learn-postgresql#94]
https://github.com/sosedoff/pgweb `|>` https://sosedoff.github.io/pgweb/

![image](https://user-images.githubusercontent.com/194400/211465050-743fca86-eeaf-431e-9658-ba49efe26bd1.png)

Looks decent. 💭

**Comment by nelsonic** [comment]
Installing _now_ with `homebrew`:

```sh
brew update
brew install pgweb
```

**Comment by nelsonic** [comment]
```sh
pgweb
```

Starts the server and opens browser:

![image](https://user-images.githubusercontent.com/194400/211559250-2faecae1-c1c5-4de6-822c-2bf128d043b5.png)

Create connection:
![image](https://user-images.githubusercontent.com/194400/211559580-2be10011-729c-4dff-b1fa-4bb6df675753.png)

**Comment by nelsonic** [comment]
Query works:

![image](https://user-images.githubusercontent.com/194400/211560936-5f3be113-b3e4-4171-b937-16f87b5595aa.png)

**Comment by ndrean** [comment]
Just saw this. Do you use it?

**Comment by nelsonic** [comment]
@ndrean yeah, been using it for a few months and _currently_ using it to write/test queries. 🔍 
Recommended.
